src/corelibs/U2Test/src/gui_tests/GUITestLauncher.cpp

```diff
@@ -229,7 +232,7 @@ bool GUITestLauncher::initTestList() {
     } else {
         // Run all tests with the given label as a single suite.
         // If label is not provided: all tests are selected.
-        testList = guiTestBase->getTests(UGUITestBase::Normal, qgetenv("UGENE_GUI_TEST_LABEL"));
+        testList = guiTestBase->getTests(UGUITestBase::Normal, labelList);
```

Comment:
Fix the comment. As I understand, tests that contain all the labels in the list will be run.

Reply:
Done